### ISSUE TITLE
Answer some problem

### ISSUE THREAD

**Issue by k1m0ch1** [fredianriko/Skripsi#1]
Hi! I can help to answer some of the problems with your beauty work.

**how to scrap and save it as CSV file or Excel File** = python have a library called `csv` just convert from Dict type to csv output file

**how to scrap tweets that contain english alphabet** = if you already understand to scrap some tweet with `tweet.text` then you just simply using library `langdetect` and call the class `detect` and you just simple `detect(tweet.text)` this will result with language code. 
here is the link https://pypi.org/project/langdetect/ 
**don't use nltk** to OP I think.

**how to scrap tweets that contain non-english alphabets (chinese character, russian alphabet,etc)** same as above I think, 

I don't get it with another question.

actually the best way to scrap, is without access to their API, try to scrap the page like this https://github.com/twintproject/twint, also I think Jupyter Notebook is too OP for this scrapper, you can do faster than that.

this is just my personal opinion, you can keep continuing your work, it is good, love your work man! 

This is a good start, but actually you can do better, here is some keyword/ link you can research more:

- OSINT ( awesome OSINT ) 
- Twitter Ego-Network ( Twitter Graph )
- https://github.com/twintproject/twint
- awesome web scraping
- https://realpython.com/beautiful-soup-web-scraper-python/
- https://twitter.com/leak_scavenger <= best OSINT bot I saw.

**Comment by fredianriko** [comment]
Hello yahya.. thank you so much for sending me an email responding to my
task.

I already using twint project to scrap tweets,Thankfully i found out about
twint on youtube channel called null byte, but i was using it inside
virtual box with Kali linux OS.. my problem was trying to scrap using
different method/script these developer created using windows 10 OS...
maybe it is just me that not looking for more information about how to run
it on windows.

apparently, i found that the problem with scraping with non-latin word is
the encoding... it all fixed when i try to tidy up my scraping result from
(csv) to (excel) format by changing the encoding to (UTF-8)... when i
change the UTF-8 the scraping result become the same as what appear on the
twitter website.

well, now my next problem is to find the right method to check hashing on
picture locally on my host with picture that i uploaded to twitter... for
this far, i already fixed my scraping method problem.

Thank you so much for your response

**Comment by k1m0ch1** [comment]
do your ever try using windows ubuntu ? I mean by WSL.
mostly python developer run on unix system, and WSL solve some of it.

**Comment by fredianriko** [comment]
yes.. i'm still don't understand much about WSL, i using it recently.. if
not mistaken using windows powershell i supposed?

i tried to use it, install twint with it, but apparently everytime i type
command after install twint it always send me error message that twint is
not recognisable...
so i move on to more fixed solution which is kali linux which i already
installed before, and i remember already installed twint in it
